.eslintrc.js

```diff
@@ -37,22 +37,13 @@ module.exports = {
     'no-unused-vars': 'off',
     'prefer-object-spread': 'off',
     'spaced-comment': 'off',
-    'prettier/prettier': [
```

Comment:
Do prettier's defaults line up with the rules that were removed below?

Reply:
I moved these rules into the new .prettierrc.json file (except for `plugins`, because that option doesn't exist and the docs say it will be auto loaded)

Reply:
Ah, I much prefer them in the ESLint config. It'd be sweet if you could move those back. Also, since we're already extending eslint-config-rollup, you're going to get some conflicts with https://github.com/rollup/eslint-config-rollup/blob/c4d81fb97d59fc1b16f209d5925001a75262babd/.eslintrc.json#L31-L38

If anything, we should be making changes there, and cleaning up the extension of that config here.

Reply:
>Also, since we're already extending eslint-config-rollup, you're going to get some conflicts

Ah yep.

I moved it to the separate file because otherwise the direct `prettier` runs (like the `prettier --check .`) won't see it 🤔

Reply:
hm, good point. maybe prettier has extendability now, I'll take a look

Reply:
https://prettier.io/docs/en/configuration.html#sharing-configurations

so what we'll want to do is add the prettier config to `eslint-config-rollup` and export that. and then use `.prettierrc.js` to consume that locally in the repo here. then I think we'll be in a manageable spot without code/config dupes.

Reply:
Sure . Or counter proposal, we remove prettier from the eslint configs completely and just use eslint for code quality, and then prettier can just do it's thing as part of the commit hook?

https://prettier.io/docs/en/comparison.html

Thoughts?

Reply:
prettier and ESLint fight each other quite a bit and I very much dislike running the two tools separately. I've found the integration of the two to be a far superior solution, so I'd like to keep it as is.

Reply:
@shellscape I opened a pr to expose the prettier config: https://github.com/rollup/eslint-config-rollup/pull/4